i18n/en.json

```diff
@@ -84,5 +84,6 @@
 	"powered-by-toolforge": "Powered by Toolforge",
 	"privacy-policy": "Privacy policy",
 	"report-issue": "Report a bug",
-	"view-source": "View on Github"
+	"view-source": "View on Github",
+	"translatewiki": "Help translate this tool on Translatewiki"
```

Comment:
I think a shorter message might be good, e.g. "Translate on translatewiki.net", so it doesn't take up too much space:

![image](https://github.com/user-attachments/assets/6c48e95e-3f80-4ba8-8598-aa07141b9106)

Note also that normal way that translatewiki.net is written is like that, all lowercase and with the ".net".